contrib/win32/win32compat/win32_pty.c

```diff
@@ -96,33 +98,62 @@ int is_conpty_supported()
 	return 0;
 }
 
-int exec_command_with_pty(wchar_t* cmd, STARTUPINFOW* si, PROCESS_INFORMATION* pi, int ttyfd)
+int exec_command_with_pty(int * pid, char* cmd, int in, int out, int err, unsigned int col, unsigned int row, int ttyfd)
 {
-	HANDLE ttyh = (HANDLE)w32_fd_to_handle(ttyfd);
+	PROCESS_INFORMATION pi;
+	STARTUPINFOW si;
 	wchar_t pty_cmdline[MAX_CMD_LEN] = { 0, };
 	int ret = -1;
+	HANDLE ttyh = (HANDLE)w32_fd_to_handle(ttyfd);
+	wchar_t * cmd_w = NULL;
+
+	if ((cmd_w = utf8_to_utf16(cmd)) == NULL)
+		return ret;
+
+	memset(&si, 0, sizeof(STARTUPINFO));
+	si.cb = sizeof(STARTUPINFO);
+	si.dwXSize = 5;
```

Comment:
Why 5 ? Add comments if needed

Reply:
remove these two lines. It was from the existing codes.